src/tests/test_config.py

```diff
@@ -284,35 +327,40 @@ def test_loading_valid_home_toml_shows_feedback_and_modifies_config(
 
 
 @pytest.mark.parametrize(
-    "file_content, param",
+    "file_content, expected_message",
     [
         (
             """
 [tool.jupysql.SqlMagic]
 """,
-            "[tool.jupysql.SqlMagic] present in {path} but empty.",
+            "[tool.jupysql.SqlMagic] present in {primary_path} but empty.",
         ),
-        ("", "Tip: You may define configurations in {path}."),
+        ("", "Tip: You may define configurations in {primary_path} or {alt_path}."),
```

Comment:
Maybe we should test the entire message displayed instead of substrings since multiple statements have been added now.

Reply:
@neelasha23 I tried doing that but it made a lot of existing tests redundant, so I decided to leave the existing tests as is.